store/extensions/[AIO] Gamba Suite/extension.json

```
@@ -0,0 +1,85 @@
+{
+  "title": "[AIO] Gamba Suite",
+  "description": "Pkr, 13/21 and Tri dice automated rolling and resetting with in-chat hand evaluation. The all-in-one dice management extension.",
+  "authors": [
+    {
+      "name": "JTD",
+      "hotel": ".com",
+      "username": "JTD",
+      "discord": "@JTD420"
```

remove the @ and only lowercase for discord usernames
```suggestion
      "discord": "jtd420"
```